opennlp-tools/src/test/java/opennlp/tools/lemmatizer/LemmatizerEvaluatorTest.java

```diff
@@ -17,24 +17,23 @@
 
 package opennlp.tools.lemmatizer;
 
-import static junit.framework.Assert.assertNotSame;
-import static org.junit.Assert.assertEquals;
+import opennlp.tools.cmdline.lemmatizer.LemmaEvaluationErrorListener;
+import opennlp.tools.util.MockInputStreamFactory;
+import opennlp.tools.util.PlainTextByLineStream;
+import org.junit.Test;
 
 import java.io.ByteArrayOutputStream;
 import java.io.IOException;
 import java.io.InputStream;
 import java.io.OutputStream;
 
-import org.junit.Test;
-
-import opennlp.tools.cmdline.lemmatizer.LemmaEvaluationErrorListener;
-import opennlp.tools.util.MockInputStreamFactory;
-import opennlp.tools.util.PlainTextByLineStream;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNotSame;
```

Comment:
I am waiting to team consensus, if teams's ok with static imports we can retain them. I personally will not keep any static imports in my code.